### PR TITLE
[dashboard] handle markdown error

### PR DESCRIPTION
**Pull request by graceguo-supercat**
### CATEGORY

Choose one

- [x] Bug Fix
- [x] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Superset Dashboard allow user to add markdown component with Markdown syntax. But user that not familiar with html may type invalid html tags that cause react throws exception and break whole page. 
For example, an airbnb user typed in `</br></br>` in markdown component, and the whole dashboard is crashed.

### BEFORE
when user open a dashboard with bad markdown component, or adding a bad markdown into dashboard in edit mode:
Screen Shot 2020-03-11 at 10 48 12 AM

### AFTER

- if user opens a dashboard with bad markdown component:
Screen Shot 2020-03-11 at 8 13 19 PM

- if user adds a bad component in edit mode:
Screen Shot 2020-03-11 at 8 16 03 PM

UMGSzj3abW

### TEST PLAN
Manual test

### REVIEWERS
@etr2460 @williaster

### CODE REVIEW

File: superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx [hunk 1]
```diff
@@ -73,6 +77,10 @@ const markdownPlaceHolder = `# ✨Markdown
 
 Click here to edit [markdown](https://bit.ly/1dQOfRK)`;
 
+const emergencyCode = `
```
Comment (mistercrunch) [inline superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx]: i18n

File: superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx [hunk 2]
```diff
@@ -73,6 +77,8 @@ const markdownPlaceHolder = `# ✨Markdown
 
 Click here to edit [markdown](https://bit.ly/1dQOfRK)`;
 
+const emergencyCode = t('This markdown component has an error.');
```
Comment (etr2460) [inline superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx]: I'd name this `errorMessage` instead

**Review by etr2460 — APPROVED** [review]
one nit, otherwise lgtm. also, could you add test plans/screenshots of the new undo/redo feature?

File: superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx [hunk 3]
```diff
@@ -65,14 +69,16 @@ const propTypes = {
...
-const markdownPlaceHolder = `# ✨Markdown
+const MARKDOWN_PLACE_HOLDER = `# ✨Markdown
 ## ✨Markdown
 ### ✨Markdown
 
 <br />
 
 Click here to edit [markdown](https://bit.ly/1dQOfRK)`;
 
+const MARKDOWN_ERROR_MESSAGE = t('This markdown component has an error.');
```
Comment (graceguo-supercat) [inline superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx]: i name rename 2 pre-defined markdown constant as `MARKDOWN_ERROR_MESSAGE` and `MARKDOWN_PLACE_HOLDER `

**Comment by graceguo-supercat** [comment]
Added a gif to show redo/undo flow to fix markdown error.

**Pull request merged**